content/docs/rfcs/37/README.md

```diff
@@ -0,0 +1,151 @@
+network peers MAY discard all [stored](https://rfc.vac.dev/spec/13/) messages addressed to the content topic `/wakunoise/1/<ct-id>/proto`. 
+In this the case and in order for parties to retrieve any eventually delayed message, 
+peers SHOULD wait a fixed amount of time before discarding stored messages corresponding to a stale Noise session. 
+
+A stale Noise session cannot be directly marked as active
+and parties are required to instantiate a new Noise session if they wish to communicate again. 
+
+However, parties can optionally include the `session-id` corresponding to a stale Noise session in the [prologue information](https://noiseprotocol.org/noise.html#prologue) employed in the Noise handshake they execute to instantiate their new Noise session. 
+This effectively emulates a mechanism to _"re-activate"_ a stale Noise session by binding it to a newly created active Noise session.
+
+In order to reduce users' metadata leakage, it is desirable (as suggested in [35/WAKU2-NOISE](https://rfc.vac.dev/spec/35/#after-handshake)) that content topics used for communications change every time a new message is exchanged.
+This can be easily realized by employing a key derivation function to compute a new `session-id` from the previously employed one (e.g. `session-id = HKDF(prev-session-id)`),
+while keeping the Inbound/outbound Cipher States, the content topic derivation mechanism and the stale mechanism the same as above.
+In this case, when one party sends **and** receives at least one message, 
+he SHALL publicly mark as stale all Noise sessions relative to messages exchanged before the earlier of these two send/receive events.
+
+# Multi-Device support
+
+Alice and Bob might possess one or more devices (e.g. laptops, smartphones, etc.) they wish to use to communicate.
+In the following we assume Alice and Bob to possess $N$ and $M$ devices, respectively.
+
+Since a Noise session contains cryptographic material required to encrypt and decrypt messages exchanged on a pre-defined content topic derived from a `session-id`, 
+messages should be encrypted and decrypted within the Noise session instantiated between the currently-in-use sender's and receiver's device.
+
+This is achieved through two main supported session management mechanisms that we called `N11M` and `NM`, respectively.
+
+## The $N11M$ session management mechanism
+
+In a $N11M$ setting, each party's device shares the same Noise session information used to encrypt and decrypt messages exchanged with the other party.
+
+![](./N11M.png)
+
+More precisely, once the first Noise session between any of Alice's and Bob's device is instantiated, 
+its session information is securely propagated to all other devices, 
+which then become able to send and receive new messages on the content topic associated to such session.
+
+The most updated session information between Alice and Bob is propagated in encrypted form to other devices, 
+using previously instantiated Noise sessions. 
```

Comment:
Do you mean `previously instantiated` or the `current`?

Reply:
Please clarify this paragraph, is is somewhat ambiguous

Reply:
I meant that when Alice instantiates a session with Bob or updates such information by simply exchanging messages (e.g. new content topic, nonces, rekeys, etc.), all Alice's devices (resp. Bob's devices) that wants to receive such updated session information, are required to have an already instantiated Noise session between them in order to receive it in encrypted form. What do you find ambiguous, exactly?

Reply:
> all Alice's devices (resp. Bob's devices) that wants to receive such updated session information, are required to have an already instantiated Noise session between them in order to receive it in encrypted form.

Thanks for the clarification, the part I quoted above is missing in the text.

Reply:
Addressed in https://github.com/vacp2p/rfc/commit/f4c6b40ed3647120770a6330355c09e854d45e0c